ddtrace/internal/wrapping.py

```diff
@@ -0,0 +1,395 @@
+import sys
+from types import FunctionType
+from typing import Any
+from typing import Callable
+from typing import Dict
+from typing import Optional
+from typing import Tuple
+from typing import cast
+
+from six import PY3
+
+
+try:
+    from typing import Protocol
+except ImportError:
+    from typing_extensions import Protocol  # type: ignore[misc]
+
+from bytecode import Bytecode
+from bytecode import Compare
+from bytecode import CompilerFlags
+from bytecode import Instr
+from bytecode import Label
+
+
+PY = sys.version_info[:2]
```

Comment:
```suggestion
from .compat import PYTHON_VERSION_INFO
```

We could use PYTHON_VERSION_INFO instead of PY